metadata-ingestion/src/datahub/sdk/_shared.py

```diff
@@ -49,6 +50,8 @@
 
 ActorUrn: TypeAlias = Union[CorpUserUrn, CorpGroupUrn]
 
+_DEFAULT_ACTOR_URN = CorpUserUrn("__ingestion").urn()
```

Comment:
not in the scope of this PR but, should this "ingestion actor" configurable?

Reply:
maybe - we don't show it anywhere in the UI, so it's not really particularly relevant. it's here mainly because a couple fields require it for backwards compat